hashring/hashring.go

```diff
@@ -76,6 +80,7 @@ func New(hashfunc func([]byte) uint32, replicaPoints int) *HashRing {
 		hashfunc: func(str string) int {
 			return int(hashfunc([]byte(str)))
 		},
+		logger: logging.Logger("ring"),
```

Comment:
I agree with what you're saying @thanodnl, but that train has left the station. This is something I wanted too, to be able to pass loggers around and set the "local" field once in one place. Unfortunately, with the current implementation, this is impossible.